src/gradualizer.erl

```diff
@@ -16,8 +16,7 @@
          type_check_dir/2
         ]).
 
--type options() :: proplists:proplist().
-
+-include("gradualizer.hrl").
```

Comment:
Use `-export_type` if we need the type in other modules. Include files are not needed for this purpose.